Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -17,8 +17,8 @@
 #include "Common/Logging/Log.h"
 #include "Core/ConfigManager.h"
 
-// This shouldn't be a global, at least not here.
-std::unique_ptr<SoundStream> g_sound_stream;
+static std::unique_ptr<Mixer> s_mixer;
```

Comment:
Would it not be better to have the `SoundStream` manage the lifetime of the mixer, so its destruction is automatically handled, but the old SoundStream simply has a means to transfer or 'hand off' its mixer over to the new one before destruction? It'd cut down on mutable static state a little.

Reply:
By "automatically handled", do you mean that instead of deleting both the mixer and the sound stream, you'd like to see it only have to delete the sound stream? I don't really see the benefit to that: the mixer, logically, isn't part of an output stream, but something that can be used separately (see #6130: there's no need to even call the Mix function).